component-test/test_dbus_proxy.py

```diff
@@ -521,7 +521,7 @@ class TestProxyMethods(object):
             "direction": "*",
             "interface": "*",
             "object-path": "*",
-            "method": ["Method1", "Method3", "Method4", "Method2"]
+            "method": ["Method1", "Method3", "Method2", "Method4"]
```

Comment:
But this test is actually examines different presentation of method argument.
The other test examines order of the methods in list presentation.